src/test/java/uk/gov/hmcts/sptribs/citizen/event/CicCreateCaseEventTest.java

```diff
@@ -55,20 +55,18 @@ public void setUp() {
 
     @Test
     void shouldChangeCaseStateWhenAboutToSubmit() {
-        // Given
-        CaseDetails<CaseData, State> details = new CaseDetails<>();
-        CaseDetails<CaseData, State> beforeDetails = new CaseDetails<>();
+        final CaseDetails<CaseData, State> details = new CaseDetails<>();
+        final CaseDetails<CaseData, State> beforeDetails = new CaseDetails<>();
+
         details.setId(TEST_CASE_ID);
-        CaseData data = caseData();
-        details.setData(data);
+        final CaseData caseData = caseData();
+        details.setData(caseData);
 
-        // When
-        AboutToStartOrSubmitResponse<CaseData, State> response = cicCreateCaseEvent.aboutToSubmit(
+        final AboutToStartOrSubmitResponse<CaseData, State> response = cicCreateCaseEvent.aboutToSubmit(
             details,
             beforeDetails
-        );
+            );
 
-        // Then
```

Comment:
A new test can be added with RepresentativeFullName and IsRepresentativePresent can be asserted for Yes.

Reply:
Created https://tools.hmcts.net/jira/browse/DTSSTCI-366 to resolve this.